benchmarks/distributed/rpc/parameter_server/launcher.py

```diff
@@ -185,22 +202,23 @@ def run_benchmark(rank, model, data, config):
         )
     else:
         # trainers = [0, trainer_count)
-        trainer_config = config.trainer_config
-        ps_config = config.ps_config
-        if (USE_CUDA_RPC in trainer_config and
-            trainer_config[USE_CUDA_RPC] and
-            USE_CUDA_RPC in ps_config and
-            ps_config[USE_CUDA_RPC] and
-                config.ps_count > 0):
-            ps_rank = get_parameter_server_rank(rank, config)
-            ps_name = get_name(
-                ps_rank,
-                config
-            )
-            rpc_backend_options.set_device_map(
-                ps_name,
-                {rank: ps_rank}
-            )
+        trainer_configs = config.trainer_config["configurations"]
+        if config.ps_count > 0:
+            ps_configs = config.ps_config["configurations"]
+            if (USE_CUDA_RPC in trainer_configs and
+                trainer_configs[USE_CUDA_RPC] and
+                USE_CUDA_RPC in ps_configs and
+                ps_configs[USE_CUDA_RPC] and
+                    config.ps_count > 0):
```

Comment:
curious, is this format consistent with PEP8/Flake8? I had an impression that it would generate sth like:

```python
if (
    USE_CUDA_RPC in trainer_configs and
    trainer_configs[USE_CUDA_RPC] and
    USE_CUDA_RPC in ps_configs and
    ps_configs[USE_CUDA_RPC] and
):
```

Reply:
yes no warnings/errors after flake8 lint. I'll update the conditional statement.